js/sign/tests/integrityblock_test.js

```diff
@@ -0,0 +1,23 @@
+import * as wbnSign from '../lib/wbn-sign.js';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as crypto from 'crypto';
+import url from 'url';
+const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
+
+describe('Integrity Block', () => {
+  function initSignerWithTestWebBundleAndKeys() {
+    const keypair = crypto.generateKeyPairSync('ed25519');
+    const file = path.resolve(__dirname, 'testdata/unsigned.wbn');
+    const contents = fs.readFileSync(file);
+    const signer = new wbnSign.IntegrityBlockSigner(contents, {
+      key: keypair.privateKey,
+    });
+    return signer;
+  }
+
+  it('Only ed25519 type of key is accepted.', () => {
```

Comment:
Test description should follow the [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development) style where method name + description read as an English sentence. e.g.
```javascript
it('accepts only ed25519 type of key.' ...)
```